### ISSUE TITLE
(Optionally) run checks after compilation

### ISSUE THREAD

**Issue by denisdefreyne** [nanoc/features#33]
After the `compile` command finishes, it might be worth running the checks, so that potential problems are surfaced as soon as possible.

**Comment by iay** [comment]
Makes sense as long as the checks run after compilation can be selected independently.

**Comment by denisdefreyne** [comment]
Yeah; these checks would be the same ones that run when calling `nanoc check` or `nanoc deploy`.

**Comment by iay** [comment]
Would the enabled checks also be run after a `nanoc live` iteration?
